docs/Developers/Development_Setup.rst

```diff
@@ -91,25 +91,21 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+and its dependencies with pip3 using
 
 ::
 
-    (coala-venv)$ git clone https://github.com/coala/coala
-    (coala-venv)$ cd coala
-    (coala-venv)$ pip3 install .
-    (coala-venv)$ cd -
-    (coala-venv)$ git clone https://github.com/coala/coala-bears
-    (coala-venv)$ cd coala-bears
-    (coala-venv)$ pip3 install .
-
-Once you have forked the repository you would like to modify, you can
-install a repository-backed version of the repository using
-
-::
-
-    (coala-venv)$ pip3 install -e <path/to/forked/repository>
+     (coala-venv)$ git clone https://github.com/coala/coala
```

Comment:
why is the indentation level on these lines being modified?
It seems to be ok with the previous indentation level.